src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -1,413 +1,169 @@
-          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
-        >
-          <AccountIcon />
-        </NavLink>
-      )
-    }
+// UniversalNavbarExpanded complex siblings
+import DropdownNav from './DropdownNav/DropdownNav'
+import MobileNav from './MobileNav/MobileNav'
 
-    const Hamburger = () => (
-      <div className={styles.hamburger}>
-        <TransformingBurgerButton
-          showMobileMenu={showMobileMenu}
-          clickHandler={this.toggleHamburger}
-          keyPressHandler={this.handleHamburgerKeyPress}
-        />
-      </div>
-    )
+// Styles
+import styles from './UniversalNavbarExpanded.module.scss'
 
-    return (
-      <div className={styles.blockNavbar}>
-        <div className={styles.navbar}>
-          <Layout.ScrollDetector>
-            <Hamburger />
-            <div
-              className={
-                showMobileMenu
-                  ? [styles.phoneAndTablet, styles.visible].join(' ')
-                  : styles.phoneAndTablet
-              }
-            >
-              <div
-                className={
-                  showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
-                }
-              >
-                <NavLink
-                  href={logoHref}
-                  LinkComponent={LinkComponent}
-                  className={styles.phoneLogo}
-                >
-                  {LogoWhite({ className: styles.logo })}
+/**
+ * Fork of UniversalNavbar for testing on CMS, likely to be consolidated
+ * into one component before deploy to Mono.
+ *
+ * Top level website navigation, fixed to the top of the viewport while scrolling.
+ * Consumers can provide a custom link and content structure.
+ *
+ * TODO for more reusability, make the Account/Search links agnostic and optional
+ *
+ * @param {boolean} hideMobileCta - Hide cta on mobile
+ * @param {boolean} hideDesktopCta - Hide cta on desktop
+ * @param {function} trackCtaClick - Analytics function run when CTA Button is clicked
+ * @param {object} LinkComponent - Agnotistic Reach and React Router Link (ex. Gatsby's <Link>)
+ * @param {string} logoHref - Href for the logo
+ * @param {object} links - URLs and text
+ *
+ * @return {JSX.Element}
+ */
+const UniversalNavbarExpanded = ({
+  LinkComponent,
+  hideMobileCta,
+  hideDesktopCta,
+  logoHref,
+  trackCtaClick,
+  links,
+}) => {
+  const BELOW_ACCORDION_LINKS = [links.CTA, links.ACCOUNT, links.SEARCH]
+
+  const SearchIconLink = () => (
+    <NavLink className={styles.searchIcon} href={links.SEARCH.href}>
+      <SearchIcon />
+    </NavLink>
+  )
+
+  const AccountIconLink = () => (
+    <NavLink className={styles.accountIcon} href={links.ACCOUNT.href}>
+      <AccountIcon />
+    </NavLink>
+  )
+
+  const layoutClasses = [styles.flex, styles.itemsCenter]
+
+  return (
+    <div className={styles.navbarWrapper}>
+      <div className={styles.navbar}>
+        <Layout.ScrollDetector>
+          <MobileNav
+            extraClass={'isFixedCta'}
```

Comment:
Is there any way to pass in the `styles.isFixedCta` directly (above comment)?

Reply:
This relates to the compiled main `.css` file that uses `FancyAnimatedLogo.scss`. Same thing I talked about IRL regarding deprecating the old navbar component and upgrading that file to `.module.scss` at a later date.